### PR TITLE
add overview section to quay navbar

### PR DESCRIPTION
**Pull request by Marcusk19**
Adds overview page to quay.io's navbar

### CODE REVIEW

File: static/beta/prod/navigation/quay-navigation.json [hunk 1]
```diff
@@ -3,6 +3,26 @@
...
+      "title": "Overview",
+      "icon": "QuayIoIcon",
+      "description": "Overview information about quay.io",
+      "href": "/quay/overview"
+      "permissions": [
+        {
+            "method": "apiRequest",
+            "args": [
+                {
+                    "url": "/beta/apps/quay/plugin-manifest.json",
```
Comment (Hyperkid123) [inline static/beta/prod/navigation/quay-navigation.json:18]: Is this still required? Is the Quay plugin still supposed to be hidden from some users? Just curious.

**Comment by Hyperkid123** [comment]
Seems like the JSON is malformed somehow
```
11:04:29 1.123 File /src/chrome-service-backend/static/beta/prod/navigation/quay-navigation.json
11:04:29 1.125 panic: invalid character '"' after object key:value pair
```

File: static/beta/prod/navigation/quay-navigation.json (hunk 1 above)
Comment (Marcusk19) [reply static/beta/prod/navigation/quay-navigation.json:18]: Not sure, I just figured it's best to leave it the same as the other navs.

Comment (Hyperkid123) [reply static/beta/prod/navigation/quay-navigation.json:18]: @Marcusk19 can I request a follow-up to this? I vaguely remember this was done during the initial release to ensure the nav items show when the deployment is ready. The quay plugins are now in production so this API check will always be true. Which makes the API calls obsolete and wastes resources.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**